src/System.IO.Compression/src/System/IO/Compression/FastEncoder.cs

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace System.IO.Compression
+{
+    internal class FastEncoder
+    {
+        private FastEncoderWindow inputWindow; // input history window
```

Comment:
Thanks Stephen - I took these all through our rewriter which I believed should have done this.  I'll do this manually and look at why it wasn't handled in the rewriter.